pkg/providers/instancetype/suite_test.go

```diff
@@ -229,12 +232,13 @@ var _ = Describe("Instance Types", func() {
 		}
 
 		// Ensure that we're exercising all well known labels except for accelerator labels
-		expectedLabels := append(v1alpha5.WellKnownLabels.Difference(sets.NewString(
-			v1alpha1.LabelInstanceAcceleratorCount,
-			v1alpha1.LabelInstanceAcceleratorName,
-			v1alpha1.LabelInstanceAcceleratorManufacturer,
-		)).UnsortedList(), lo.Keys(v1alpha5.NormalizedLabels)...)
-		Expect(lo.Keys(nodeSelector)).To(ContainElements(expectedLabels))
+		Expect(lo.Keys(nodeSelector)).To(ContainElements(
+			append(
+				v1alpha5.WellKnownLabels.Clone().Delete(v1.LabelWindowsBuild).Difference(sets.NewString(
```

Comment:
I think you might be able to throw this in the bucket with the other labels like 

```go
expectedLabels := append(v1alpha5.WellKnownLabels.Difference(sets.NewString(
			v1alpha1.LabelInstanceAcceleratorCount,
			v1alpha1.LabelInstanceAcceleratorName,
			v1alpha1.LabelInstanceAcceleratorManufacturer,
                         v1.LabelWindowsBuild,
		)).UnsortedList(), lo.Keys(v1alpha5.NormalizedLabels)...)
```

Reply:
I now understand the “Difference” method now. :)
Will fix the test code.